### DISCUSSION TITLE
Linked simulations in APSIMX

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9299] [Q&A] [answered]
Why is there no linked function in APSIMX? I remember this function is frequently being used in Classic.

**Comment by hut104** [comment, marked as answer]
APSIM Classic had "ShortCuts" to link elements of simulations so that they were the same.  Whilst powerful, there were often mistakes by users with links to incorrect nodes.  The code was also complex handle the many impacts of such a design.

In Next Generation, we use experiments.  Experiments can be used to do the same thing, but are much safer and even more powerful.  Simulation experiments of up to millions of permutations can be achieved safely.

**Reply by KeLiu7** [reply]
Thanks Neil.
